Live/LiveEventWithDVR/README.md

```diff
@@ -1,29 +1,31 @@
-# Live event with DVR
+# Live Event with DVR
 
-This sample demonstrates how to create and use LiveEvents and LiveOutputs in the v3 Media Services API. It performs the following tasks:
+This sample demonstrates how to create and use Live Events and Live Outputs in the v3 Media Services API. It performs the following tasks:
 
 1. Creates a "standard" pass-through LiveEvent.
```

Comment:
Wonder if we should update this sample to actually be a transcoding live event and not a pass-through?  I would think our primary use case would be for single bitrate input and transcoding output?  I believe we initially made this a pass through to keep it low cost.  If we wanted to keep it as a pass through sample, then we should really make it a Basic pass through to keep the costs at 0.20/hour.

Reply:
I think that can be a separate change.